README.md

```diff
@@ -95,117 +98,206 @@ your system's `$PATH`.
 > ./install.sh
 > ```
 
-#### Manual build
+# 🐳 Docker
 
-Clone this repository. Then, using the Go tool-chain, you can build a binary:
+Use the container image of the CLI for use in CI/CD or automation:
+
+```sh
+$ docker run ghcr.io/open-sauced/pizza-cli:latest
+```
+
+For commands that require access to your file system (like `generate codeowners`), ensure
+you pass a volume to the docker container:
 
+```sh
+$ docker run -v /local/path:/container/path ghcr.io/open-sauced/pizza-cli:latest \
+    generate codeowners /container/path
 ```
-$ go build -o build/pizza main.go
+
+For example, to mount your entire home directory (which may include a `~/.sauced.yaml` file
+alongside the project you want to generate a `CODEOWNERS` file for):
+
+```sh
+$ docker run -v ~/:/app ghcr.io/open-sauced/pizza-cli:latest \
+    codeowners /app/workspace/gopherlogs -c /app/.sauced.yaml
 ```
 
-Warning! There may be unsupported features, breaking changes, or experimental
-patches on the tip of the repository. Go and build with caution!
+# 🍕 Pizza Action
+
+Use [the Pizza GitHub Action](https://github.com/open-sauced/pizza-action) for running `pizza` operations in GitHub CI/CD,
+like automated `CODEOWNERS` updating and pruning:
+
+```yaml
+jobs:
+  pizza-action:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pizza Action
+        uses: open-sauced/pizza-action@v2
+        with:
+          # Optional: Whether to commit and create a PR for "CODEOWNER" changes
+          commit-and-pr: "true"
+          # Optional: Title of the PR for review by team
+          pr-title: "chore: update repository codeowners"
+```
+
+# 📝 Docs
+
+- [Pizza.md](./docs/pizza.md): In depth docs on each command, option, and flag.
+- [OpenSauced.pizza/docs](https://opensauced.pizza/docs/tools/pizza-cli/): Learn
+  how to use the Pizza command line tool and how it works with the rest of the OpenSauced
+  ecosystem.
 
 # ✨ Usage
 
 ## Codeowners generation
 
-Use the `codeowners` command to generate an `OWNERS` file or GitHub style `CODEOWNERS` file.
+Use the `codeowners` command to generate a GitHub style `CODEOWNERS` file or a more agnostic `OWNERS` file.
 This can be used to granularly define what experts and entities have the
 most context and knowledge on certain parts of a codebase.
 
+```sh
+pizza generate codeowners /path/to/local/git/repo
+```
+
+It's expected that there's a `.sauced.yaml` config file in the given path or in
```

Comment:
consider moving this paragraph and the generate config command before the `pizza generate codeowners` command.

Reply:
Yeah, I'd considered doing that: difficult to communicate the need for a config if you aren't sure what it's for or how it'll be used. I'll move it up!